### ISSUE TITLE
Feature Request: Windows build. 

### ISSUE THREAD

**Issue by vico93** [ankurk91/google-chat-electron#21]
Is a Windows build feasible for the project? Although we have Gmail-desktop already, a dedicated Google chat standalone app would be very interesting...

**Comment by ankurk91** [comment]
Yes, it is possible.
And I don't want to use electron-builder. 

But i dont have time to build and install and test. 
I haven't used Windows from 5 years. 
Need to download the ISO and install in a VM. 😅

I am very busy with my employer projects now a days. 
I will try to extract time but can't promise before Jan 2022.

**Comment by dtolb** [comment]
@vico93 

If you can pull down locally: add the following the package.json

```
"scripts": {
    "prepack:windows": "yarn run clean:dist && yarn run ts",
    "pack:windows": "electron-packager . --out=./dist/ --icon=./resources/icons/normal/mac.icns",
    "windows": "yarn run prepack:windows && yarn run pack:windows"
},
  "devDependencies": {
    "electron-installer-windows": "^3.0.0",
  },
```

and remove
```
  "devDependencies": {
    "electron-installer-debian": "^3.1.0",
  },
```

You can then install by:

```
> npm install
> npm run windows
>
```

Then in the /dist folder you should have a workable app. I've not had any issues and follow the upstream and just rebase my changes on top each release.

![image](https://user-images.githubusercontent.com/6805534/132925093-81c8834d-f3f2-4c18-a920-4f4601abcd9d.png)

**Comment by ankurk91** [comment]
I was unable to make installer via `electron-installer-windows` on window 7

**Comment by ankurk91** [comment]
Fixed in 2.8.0
